src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [one, two] = [Big(numberOne), Big(numberTwo)];
+
+  switch (operation) {
+    case '+':
+      return one.plus(two);
+    case '-':
+      return one.minus(two);
+    case 'x':
+      return one.times(two);
+    case '/':
+      return one.div(two);
```

Comment:
Dont forget to handle DIVISION by ZERO on milestone 5 else, it will generate an error

Reply:
Thank you @Shloch. This is a good suggestion.